### ISSUE TITLE
org.springframework.boot.context.web.ErrorPageFilter needs public accessibility

### ISSUE THREAD

**Issue by davison** [spring-projects/spring-boot#2026]
There seem to be a couple of edge cases where the package private accessibility of ErrorPageFilter causes problems in an application.  While not necessarily a SpringBoot fault, containers such as Tomcat and JBoss will try and fail to access the class under certain conditions.  See http://stackoverflow.com/questions/27189047/springboot-webapp-under-java-securitymanager-throws-exceptions-when-granted-allp and https://github.com/spring-projects/spring-boot/issues/933 for more info.

It would be helpful if the class could be made public.  Thanks.
